edx-platform/pearson-certiport-theme/lms/templates/ccx/enrollment.html

```diff
@@ -3,19 +3,11 @@
 from django.utils.translation import ugettext as _
 from django.utils.translation import pgettext
 from openedx.core.djangolib.markup import HTML, Text
-from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangoapps.plugins.plugins_hooks import run_extension_point
 %>
 
 <%
-    allow_ccx_unenroll = True
-
-    if configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False) and not configuration_helpers.get_value('ALLOW_CCX_UNENROLL', True):
-        try:
-            from pearson_course_operation.utils import is_ccx_license_managed
-
-            allow_ccx_unenroll = not is_ccx_license_managed(course.id)
-        except ImportError:
-            pass
+    allow_ccx_unenroll = False if run_extension_point('PCO_IS_LICENSED_CCX', course_id=course.id) else True
```

Comment:
```suggestion
    allow_ccx_unenroll = not run_extension_point('PCO_IS_LICENSED_CCX', course_id=course.id)
```
Wouldn't it be better to use `not`?